internal/service/ses/ses_cluster.go

```diff
@@ -82,13 +82,24 @@ func ResourceNcloudSESCluster() *schema.Resource {
 							Required: true,
 						},
 						"user_name": {
-							Type:             schema.TypeString,
-							Required:         true,
-							ValidateDiagFunc: validation.ToDiagFunc(validation.StringLenBetween(3, 15)),
-						},
-						"user_password": {
 							Type:     schema.TypeString,
 							Required: true,
+							ValidateDiagFunc: validation.ToDiagFunc(validation.All(
+								validation.StringLenBetween(3, 15),
+								validation.StringMatch(regexp.MustCompile(`^[a-z]+[a-z0-9-]+[a-z0-9]$`), "Allows only lowercase letters(a-z), numbers, hyphen (-). Must start with an alphabetic character, must end with an English letter or number"),
+							)),
+						},
+						"user_password": {
+							Type:      schema.TypeString,
+							Required:  true,
+							Sensitive: true,
+							ValidateDiagFunc: validation.ToDiagFunc(validation.All(
+								validation.StringLenBetween(8, 20),
+								validation.StringMatch(regexp.MustCompile(`[a-zA-Z]+`), "Must have at least one alphabet"),
+								validation.StringMatch(regexp.MustCompile(`\d+`), "Must have at least one number"),
+								validation.StringMatch(regexp.MustCompile(`[~!@#$%^*()\-_=\[\]\{\};:,.<>?]+`), "Must have at least one special character"),
+								validation.StringMatch(regexp.MustCompile(`^[^&+\\"'/\s`+"`"+`]*$`), "Must not have ` & + \\ \" ' / and white space."),
+							)),
```

Comment:
it might be good to separate and define a new validation function for the user_password to reduce code duplication. (optional)

Reply:
Looks like this should be a separate PR.